src/test/java/com/meilisearch/integration/SettingsTest.java

```diff
@@ -101,6 +104,27 @@ public void testUpdateSettingsSort() throws Exception {
         assertEquals(2, newSettings.getSortableAttributes().length);
     }
 
+    @Test
+    @DisplayName("Test update settings changing the typo tolerance")
+    public void testUpdateSettingsTypoTolerance() throws Exception {
+        Index index = createIndex("updateSettingsSynonyms");
+        Settings settings = index.getSettings();
+
+        TypoTolerance typoTolerance = new TypoTolerance();
+        typoTolerance.setEnabled(true);
+        typoTolerance.setDisableOnWords(new String[] {"and"});
+        typoTolerance.setDisableOnAttributes(new String[] {"title"});
+        settings.setTypoTolerance(typoTolerance);
+
+        index.waitForTask(index.updateSettings(settings).getUid());
+
+        Settings newSettings = index.getSettings();
+
+        assertEquals(1, newSettings.getTypoTolerance().getDisableOnWords().length);
```

Comment:
I think you should assert by the value here because if the `getDisableOnWords` responds with a `["other"]` your test will not break.